src/video/scalebit.c

```diff
@@ -50,7 +50,7 @@
 static inline void stage_scale2x(void* dst0, void* dst1, const void* src0, const void* src1, const void* src2, unsigned pixel, unsigned pixel_per_row)
 {
 	switch (pixel) {
-#if defined(__x86_64__) || defined(_M_X64) || defined(__SSE2__) || (defined(_M_IX86_FP) && (_M_IX86_FP == 2))
+#if defined(__x86_64__) || defined(_M_X64) || defined(__SSE2__) || (defined(_M_IX86_FP) && (_M_IX86_FP == 2) && defined(WITH_SSE2))
```

Comment:
Currently it seems it was only this function (I've got VS2013 Community, **v12** release: I cannot assess if it happens even on VS2012). Anyway I tested all filters (nearest, scale2x & hqx) both debug/release compilations, and no other compiler/optimizer bugs detected.
Now it's good for FreeBSD compiling? Can we accept this PR?